src/EditorFeatures/Core/BackgroundWorkIndicator/WpfBackgroundWorkIndicatorFactory.cs

```diff
@@ -39,15 +39,9 @@ IBackgroundWorkIndicatorContext IBackgroundWorkIndicatorFactory.Create(
         _threadingContext.ThrowIfNotOnUIThread();
 
         // If we have an outstanding context in flight, cancel it and create a new one to show the user.
-        _currentContext?.CancelAndDispose();
+        _currentContext?.Dispose();
 
-        // Create the indicator in its default/empty state.
-        _currentContext = new BackgroundWorkIndicatorContext(
-            this, textView, applicableToSpan, description,
-            cancelOnEdit, cancelOnFocusLost);
-
-        // Then add a single scope representing the how the UI should look initially.
-        _currentContext.AddScope(allowCancellation: true, description);
+        _currentContext = new BackgroundWorkIndicatorContext(this, textView, applicableToSpan, description, cancelOnEdit, cancelOnFocusLost);
```

Comment:
>_currentContext [](http://example.com/codeflow?start=8&length=15)

It doesn't look like OnContextDisposed is being called anymore either?

Reply:
it def is called.  crazy tod..

i certainly didn't make a mistake and forget to call it.  nosiree.